includes/class-wc-stripe-account.php

```diff
@@ -403,7 +403,7 @@ public function is_webhook_enabled() {
 
 			return 'enabled' === $webhook_status;
 		} catch ( Exception $e ) {
-			WC_Stripe_Logger::log( 'Unable to determine webhook status: .;' . $e->getMessage() );
+			WC_Stripe_Logger::error( 'Unable to determine webhook status: .;' . $e->getMessage() );
```

Comment:
This message looks like it could be cleaned up and possibly have the exception message shifted to the context.
```suggestion
			WC_Stripe_Logger::error( 'Unable to determine webhook status: ' . $e->getMessage() );
```